### ISSUE TITLE
[Python]Seek for help on extracting certain content according to font and paragragh setting in docx documents

### ISSUE THREAD

**Issue by charlessws1985** [donnemartin/system-design-primer#535]
I'm a primary learner on Python. My request is to extracting certain content according to font and paragragh setting in docx documents. Meanwhile the content I want to extract have nothing to do with the style. (For example. In one docx document all the words are of Normal style. But some words are "Batang" font and others are "Arial" font. I need to extract words of "Arial" font).

After 2 days searching I found most third party modules including Python-docx don't have the function. 
I would be much appreciate if any master could help.

**Comment by CJL-3333** [comment]
You are correct that python-docx does not expose a direct .font property on the paragraph level because a single paragraph can contain multiple fonts. To extract text based on font name (like "Arial" vs. "Batang"), you need to look at the runs within the paragraphs.
